migrator/README.md

```diff
@@ -52,7 +52,16 @@ In case of a successful run, the script prints
 If you want to get some more details on the actual migration steps, enable logging while performing the database migration:
 
     ./bin/ups-migrator --logLevel=DEBUG update
+    
+#### Running the script from source
 
+If you are developing UnifiedPush Server, you can run the script from source instead of from distribution:
+
+    cd aerogear-unifiedpush-server/migrator/
+    mvn clean install
+    mvn dependency:copy-dependencies -DincludeScope=runtime
```

Comment:
Would it be worth having a a profile that does this, like ```mvn -PsetupEnv``` or something like that?